tests/src/main/java/net/neoforged/neoforge/debug/client/TextureAtlasTests.java

```diff
@@ -60,4 +72,52 @@ static void testMaterialAtlas(final DynamicTest test) {
             });
         });
     }
+
+    @TestHolder(description = { "Tests that custom sprite metadata sections get passed through resource reloading properly" }, enabledByDefault = true)
+    @GameTest
+    @EmptyTemplate
+    static void defaultSpriteMetadataSections(final DynamicTest test) {
+        String modId = test.createModId();
+
+        var testResource = ResourceLocation.fromNamespaceAndPath(modId, "resource");
+        var sectionType = new MetadataSectionType<>("default_metadata_test", Codec.BOOL);
+
+        test.framework().modEventBus().addListener(AddPackFindersEvent.class, event -> {
```

Comment:
Why even bother with another pack finder? Could just use the normal resources folder.

Reply:
I didn't want to add the sprite source if we weren't running the test, since it adds things to the texture atlas

Reply:
Consider that all test textures add things to the atlas. It's pretty harmless :P

Reply:
Agreed, using a custom pack finder achieves nothing (avoiding stitching a single 16x16 sprite provides no benefit whatsoever) and only bloats the test